### DISCUSSION TITLE
What is rank task?

### DISCUSSION THREAD

**Discussion by hanhanwu** [microsoft/FLAML#183] [Q&A]
Quick question FLAML team, I just saw this rank task example:
![rank_task](https://user-images.githubusercontent.com/4024769/131791655-ff4a682b-eb94-4f1e-b236-139d7b430cd3.PNG)

May I ask what does this type of task do?

**Comment by sonichi** [comment]
It is for learning to rank. For example: https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.LGBMRanker.html

**Reply by hanhanwu** [reply]
Oh, it's relevant to search and rank, which is what I was hoping for. It's relevant to a new project I'm going to work on. Thank you very much!
